pkg/apis/core/v1beta1/types.go

```diff
@@ -152,14 +152,35 @@ type DataVolumeSourceS3 struct {
 
 // DataVolumeSourceRegistry provides the parameters to create a Data Volume from an registry source
 type DataVolumeSourceRegistry struct {
-	//URL is the url of the Docker registry source
+	//URL is the url of the registry source (starting with the scheme: docker, oci-archive, imagestream)
 	URL string `json:"url"`
+	//PullMethod can be either "pod" (default import), or "node" (node docker cache based import)
+	PullMethod RegistryPullMethod `json:"pullMethod,omitempty"`
```

Comment:
This should probably be a pointer if optional

Reply:
The following 2 fields (SecretRef, CertConfigMap) are also optional but not pointers, so why is it important here?

Reply:
IMO the others should be pointers as well not sure why they're not.  Fine if you want to keep non pointers for consistency for now.  Can always change to pointers while maintaining API compatibility

Reply:
Fixed